go.mod

```diff
@@ -1,6 +1,7 @@
 module contrib.go.opencensus.io/exporter/prometheus
 
 require (
-	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829 // TODO: change the version to v0.9.2 once oc-go version is updated.
```

Comment:
What was the reason for this pin to that specific version?

Reply:
https://github.com/census-instrumentation/opencensus-service/pull/512#discussion_r271958614.